src/index.html

```diff
@@ -3,11 +3,310 @@
+              <img
+                src="images/photos/picture-4.jpg"
+                class="pictures__image"
+                alt="blue speaker"
+              >
+            </div>
+          </div>
+          <p class="article__text">
+            Speakers
+          </p>
+        </article>
+
+        <article class="article section__article">
+          <div class="pictures article__pictures">
+            <div class="pictures__item pictures__item--small">
+              <img
+                src="images/photos/picture-5.jpg"
+                class="pictures__image"
+                alt="man with sunglasses"
+              >
+            </div>
+
+            <div class="pictures__item pictures__item--big">
+              <img
+                src="images/photos/picture-6.jpg"
+                class="pictures__image"
+                alt="woman with sunglasses"
+              >
+            </div>
+          </div>
+          <p class="article__text">
+            Audio sunglasses
+          </p>
+        </article>
+
+        <a href="#" class="button button--primary">View all</a>
+      </section>
+
+      <section class="section main__section" id="how-to-buy">
+        <div class="grid__buy">
+          <h2 class="title title--section main__title main__title--customer">
+            Why buy direct
+            <br>
+            from Bose
+          </h2>
+          <div class="main__content">
+            <p class="main__paragraph main__paragraph--bold">
+              Free 2-day shipping and returns
+            </p>
+            <p class="main__paragraph main__paragraph--bold">
+              90-day risk-free trial
+            </p>
+            <p class="main__paragraph main__paragraph--bold">
+              World class customer service
+            </p>
+            <p class="main__paragraph main__paragraph--bold">
+              My Bose account management
+            </p>
+            <p class="main__paragraph">A great product is more than what’s in the box. It’s a promise of premium performance, world-class support, and everything you expect from a trusted brand. It’s just one of many reasons why you’ll shop with confidence on Bose.com.</p>
+          </div>
+        </div>
+      </section>
+
+      <section class="section main__section" id="contact-us">
+        <h2 class="title title--section section__title">Contact us</h2>
+        <div class="contact">
+          <form class="form contact__form">
+            <div class="form__item">
+              <label for="name" class="form__label">Name</label>
+              <input
+                id="name"
+                name="name"
+                class="form__input"
+                type="text"
+                placeholder="Name"
+              >
+            </div>
+
+            <div class="form__item">
+              <label for="email" class="form__label">Email</label>
+              <input
+                id="email"
+                name="name"
```

Comment:
This name should reflect the value of the input:
```suggestion
                name="email"
```